### PR TITLE
Remove old post_processing example

### PR DESCRIPTION
**Pull request by IceSentry**
# Objective

- The old post processing example doesn't use the actual post processing features of bevy. It also has some issues with resizing. It's also causing some confusion for people because accessing the prepass textures from it is not easy.
- There's already a render to texture example
- At this point, it's mostly obsolete since the post_process_pass example is more complete and shows the recommended way to do post processing in bevy. It's a bit more complicated, but it's well documented and I'm working on simplifying it even more

## Solution

- Remove the old post_processing example
- Rename post_process_pass to post_processing

## Reviewer Notes
The diff is really noisy because of the rename, but I didn't change any code in the example.

### CODE REVIEW

**Review by JMS55 — APPROVED** [review]

**Review by nicopap — APPROVED** [review]
LGTM. For future reference: I think it's possible to do the change in two commit to avoid the git mixup.

File: examples/shader/post_processing.rs [hunk 1]
```diff
@@ -1,188 +1,392 @@
...
-        .add_systems(Update, main_camera_cube_rotator_system)
+        .add_systems(Update, (rotate, update_settings))
         .run();
 }
 
-/// Marks the first camera cube (rendered to a texture.)
-#[derive(Component)]
-struct MainCube;
+/// It is generally encouraged to set up post processing effects as a plugin
+struct PostProcessPlugin;
```
Comment (james7132) [inline examples/shader/post_processing.rs:48]:
```suggestion
struct PostProcessPlugin;

```

Comment (IceSentry) [reply examples/shader/post_processing.rs:48]: I'm not sure I understand what the suggestion does. It just looks like nothing in the preview.

Comment (IceSentry) [reply examples/shader/post_processing.rs:48]: Oh, it's adding a newline

**Comment by mockersf** [comment]
Could you also rename the shader `post_process_pass.wgsl` to the new example name?

**Comment by IceSentry** [comment]
Done!

**Review by mockersf — APPROVED** [review]

**Pull request merged**